ramalama

```diff
@@ -68,9 +68,9 @@ def run_curl_command(args, filename):
             sys.exit(e.returncode)
 
 
-def pull_ollama_manifest(ramalama_store, manifests, accept, registry_head, model_tag):
+def pull_ollama_manifest(repos_ollama, manifests, accept, registry_head, model_tag):
```

Comment:
Why not just call it repos.  No need to keep repeating _ollama

Reply:
My thinking was shortly I will bring in hugging face support again and there will likely be "repos_hf", "repos_ollama" directory is:

~/.local/share/ramalama/repos/ollama

shortly there will be:

~/.local/share/ramalama/repos/huggingface

Reply:
I think you should embed the location of the store inside of the functions, and not force it to be different.

Once we have multiple pull_manifests() functions, then we can start to consildate functionality and get as much reuse as possible.